src/components/tabs/__internal__/tab-title/tab-title.style.js

```diff
@@ -303,31 +300,40 @@ const StyledTabTitle = styled.li`
   height: 100%;
   position: relative;
 
-  ${({ position, borders, noRightBorder, noLeftBorder }) => `
-      ${
-        position === "top" &&
-        css`
-          ${borders &&
-          !(noRightBorder || noLeftBorder) &&
-          css`
-            &:not(:first-of-type) {
-              margin-left: -1px;
-            }
-          `}
-        `
-      }
-      ${
-        position === "left" &&
-        css`
-          ${borders &&
-          css`
-            &:not(:first-of-type) {
-              margin-top: -1px;
-            }
-          `}
-        `
-      }
+  ${({
+    position,
+    borders,
+    noRightBorder,
+    noLeftBorder,
+    warning,
+    info,
+    theme,
+  }) => css`
+    ${position === "top" &&
+    css`
+      ${borders &&
+      !(noRightBorder || noLeftBorder) &&
+      css`
+        &:not(:first-of-type) {
+          margin-left: -1px;
+        }
+      `}
+
+      ${(warning || info) &&
+      css`
+        background-color: ${theme.colors.white};
```

Comment:
I'm not sure if this will create issues for any projects not using a white background app colour. Also when a tab without an error is no longer selected it will have a transparent background

![image](https://user-images.githubusercontent.com/44157880/136053077-b6f5be03-8fdd-4ef1-bfcc-4a21a0923727.png)


What master does:
![image](https://user-images.githubusercontent.com/44157880/136052789-d2a9d542-152f-4a9e-bfb2-648e8870f990.png)